docs/docs/snippets/clients/providers/gemini.mdx

```diff
@@ -88,4 +88,4 @@ client<llm> MyClient {
 </ParamField>
 
 
-For all other options, see the [official Google Gemini API documentation](https://ai.google.dev/api/rest/v1/models/generateContent).
+For all other options, see the [official Google Gemini API documentation](https://ai.google.dev/api/rest/v1beta/models/generateContent).
```

Comment:
is v1beta a superset to v1?

Reply:
Yes, at least it is currently! I haven't found that officially stated in my (limited) review of their docs but if you list the models in each v1beta has all the models in v1 and more.

Reply:
good enough for me. I think noting that the python library does this is a good point.